src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -134,7 +134,10 @@ function paginatePages(page, files, field, layout, ariaLabel, perPage) {
 
 // Return page object with path, breadcrumb and title set.
 function updateEntityUrlObj(page, drupalPagePath, title) {
-  const pathSuffix = title.replace(/\s+/g, '-').toLowerCase();
+  const pathSuffix = title
```

Comment:
If we keep tweaking this, it might make sense to use the `words` function from lodash: `_.words(title).join('-')`, or find a module for it.